packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -135,6 +142,38 @@ const TradingAppCard = ({
                             text={badge_text}
                         />
                     )}
+                    {is_open_order_position_status_present && (
+                        <StatusBadge
+                            className='trading-app-card__acc_status_badge'
+                            account_status={'open-order-position'}
+                            icon={'IcAlertWarning'}
+                            text={
+                                <Localize
+                                    i18n_default_text='<0>{{status_text}}</0>'
+                                    values={{ status_text }}
+                                    components={[
+                                        <Text
+                                            key={0}
+                                            weight='bold'
+                                            size='xxxs'
+                                            color='var(--status-warning)'
```

Comment:
can you check if this coloring is really working? 🌈 
`var(--status-warning)` will be appended with another string inside `Text` component i believe.

Reply:
Actually the coloring does work. If you check inside `getStatusBadgeConfig` you can find the same thing. I will still change it to color='warning' which makes more sense and is the same color as var(--status-warning).